compatibility_lib/compatibility_lib/compatibility_store.py

```diff
@@ -416,6 +422,26 @@ def save_compatibility_statuses(
                 self._release_time_table,
                 row)
 
+    def _should_update_dep_info(self, cs, dep_info_stored):
+        """Return True if the stored version is behind latest version."""
+        if dep_info_stored is None:
+            return True
+
+        install_name = cs.packages[0].install_name
+        install_name_sanitized = install_name.split('[')[0] \
+            if '[' in install_name else install_name
```

Comment:
if there is no '[' in the `install_name`, `install_name.split('[')` will return a list of size 1 containing `install_name`, so I think this line is redundant

Reply:
Removed.